eng/versioning/external_dependencies.txt

```diff
@@ -1,45 +1,68 @@
-com.azure:azure-core
-com.azure:azure-core-experimental
-com.azure:azure-core-http-netty
-com.azure:azure-core-management
-com.azure:azure-core-test
-com.azure:azure-identity
-com.azure:azure-json
-
-com.fasterxml.jackson.core:jackson-core
-com.fasterxml.jackson.core:jackson-databind
-com.fasterxml.jackson.dataformat:jackson-dataformat-xml
-org.slf4j:slf4j-api
-org.slf4j:slf4j-nop
-org.slf4j:slf4j-simple
-junit:junit
-org.junit.jupiter:junit-jupiter-api
-org.junit.jupiter:junit-jupiter-engine
-org.junit.jupiter:junit-jupiter-params
-io.projectreactor:reactor-test
-
-org.apache.maven.plugins:maven-antrun-plugin
-org.apache.maven.plugins:maven-assembly-plugin
-org.apache.maven.plugins:maven-checkstyle-plugin
-org.apache.maven.plugins:maven-clean-plugin
-org.apache.maven.plugins:maven-compiler-plugin
-org.apache.maven.plugins:maven-eclipse-plugin
-org.apache.maven.plugins:maven-enforcer-plugin
-org.apache.maven.plugins:maven-failsafe-plugin
-org.apache.maven.plugins:maven-help-plugin
-org.apache.maven.plugins:maven-install-plugin
-org.apache.maven.plugins:maven-jar-plugin
-org.apache.maven.plugins:maven-javadoc-plugin
-org.apache.maven.plugins:maven-jxr-plugin
-org.apache.maven.plugins:maven-plugin-plugin
-org.apache.maven.plugins:maven-project-info-reports-plugin
-org.apache.maven.plugins:maven-release-plugin
-org.apache.maven.plugins:maven-resources-plugin
-org.apache.maven.plugins:maven-shade-plugin
-org.apache.maven.plugins:maven-site-plugin
-org.apache.maven.plugins:maven-source-plugin
-org.apache.maven.plugins:maven-surefire-plugin
+# Format;
```

Comment:
I wonder if we need this for autorest as we don't ship these libraries to maven central. We could just use Maven's `<dependencyManagement>` to maintain the versioning in the root pom and the children could just add the group/artifact in `<dependencies>`. Kind of similar to BOM but for multi-module project.

Reply:
Not sure whether we'd like to maintain another file like this.

Should the script just copy the version_client and external_dependencies from SDK repo, and concat them?